assets/src/scss/pages/search/_search-results.scss

```diff
@@ -181,7 +181,9 @@
     font-size: 1.625rem;
   }
 
-  &:visited {
+  &:visited,
```

Comment:
Out of all places where we would remove the visited state, the search results seem to be the one where such a state has the most value. For example Google search results. I don't see why we would keep an visited color in regular page links but remove it here.

Reply:
That's a conversation to have with the design team 🤷‍♀️ But I think this is for consistency, since other links that look this way on the site (such as Columns block headers and Articles block titles) have this behaviour.

Reply:
That's my assumption too. But @Inwerpsel feel free to leave a comment to the ticket for the Design team to see it. We can always re-visit later or revert that.

Reply:
Yeah, didn't specify but it was indeed not a remark for these changes, just something that struck me.